app/controllers/manage/projects_controller.rb

```diff
@@ -114,6 +114,15 @@ def project_params
       :no_cats,
       :no_dogs,
       :has_smoke_in_home,
+      :joann_helped,
```

Comment:
I should have mentioned this before, but booleans would be nice to have an "is" as a prefix if possible.  Would that work with these?  I can stretch to other prefixes, too, but mainly it's to help devs know what type of data is there.  And, with booleans, we often do the instance method with the "?", so something like "is_privacy_needed" reads pretty nicely.

Reply:
I'll keep this in mind going forward. As discussed on slack, we're going to leave this as is for the moment because there are other inconsistencies in boolean field naming in the app.